.goreleaser.yml

```diff
@@ -51,6 +51,10 @@ env:
 snapshot:
   name_template: "{{ .Tag }}-next"
 
+docker_signs:
```

Comment:
from my limited goreleaser knowledge, I think:
- We need to add signatures of blobs (not of images) to the release and/or archive, see [here](https://goreleaser.com/customization/sign/#with-cosign).
- Possibly, this `docker_signs` needs to be called after the artifacts are built, and not before.
- It may be possible to save `cosign.key` as a GHA secret, and pass it to goreleaser in an env var.

Reply:
I do not believe that the order of clauses in the .goreleaser.yml matters to execution. In the docs about `docker_signs` it says: 

> Note that this pipe will run only at the end of the GoReleaser execution (in its publishing phase), as cosign will change the image in the registry.

IOW we do not specify order, goreleaser does.

GHA secret ... Looked around, found https://docs.github.com/en/actions/security-guides/encrypted-secrets#storing-base64-binary-blobs-as-secrets 
So, looks to be possible, and will go for that. This also means creation of a new key pair to be sure. Password will stay the same, as it was never exposed.

Will look into the blob signing as well (epinio binary).

Reply:
Redone completely per https://blog.goreleaser.com/goreleaser-and-software-supply-chain-security-e8917b26924b
Commit 3659103fdbf6eff14300b838cf8e22ab7bb7e6cf